### DISCUSSION TITLE
server: ability to delete account

### DISCUSSION THREAD

**Discussion by theel0ja** [asciinema/discussions#199] [Ideas]
I think this is an essential feature.

Seems to be required by GDPR, as well:
https://gdpr-info.eu/art-17-gdpr/

**Comment by ku1ik** [comment]
Good idea 👍

**Comment by ananthb** [comment]
I need this feature too.

**Comment by ku1ik** [comment]
FYI this has been implemented in asciinema server a while ago. There's account deletion option in user settings.
